src/website/views/custom-elements/spinner.pug

```diff
@@ -0,0 +1,82 @@
+---
+title: Spinner
+scripts:
+  - js/ce/ux-chi-ce
+---
+
+p.-text Spinners are animated circles that let users know when information is saving or loading.
+
+h3 Examples
+p.-text To render a spinner, use <code>chi-spinner</code> tag.
+
+h3 Backdrop
+p.-text Wrap spinners in <code>a-backdrop</code> to overlay content similar to a modal.
+h4 Base
+.example.-mb--3
+  .a-backdrop.-p--6(style="position:relative;height:25rem;")
+    chi-spinner(color='inverse', size='lg')
+  :code(lang="html")
+    <div class="a-backdrop">
+      <chi-spinner color="inverse"></chi-spinner>
+    </div>
+
+h4 Inverse
+p.-text Apply the class <code>-inverse</code> to render spinners on light backgrounds.
+.example.-mb--3
+  .a-backdrop.-inverse.-p--6(style="position:relative;height:25rem;")
+    chi-spinner(size='lg', color='primary')
+  :code(lang="html")
+    <div class="a-backdrop -inverse">
+      <chi-spinner></chi-spinner>
+    </div>
+
+h3 Contextual
```

Comment:
This section has no content, and will not have until buttons are merged, please remove it for now.